content/terraform/v1.12.x/docs/language/functions/title.mdx

```diff
@@ -7,14 +7,18 @@ description: |-
 
 # `title` Function
 
-`title` converts the first letter of each word in the given string to uppercase.
+`title` converts the first letter of each word in the given string to uppercase.  It does not downcase any letters in the string.
```

Comment:
Hello @bschonec! Great catch and thank you for the contribution! How would you feel about rewording this a little:
```suggestion
`title` converts the first letter of each word in the given string to uppercase. The `title` function does not change any other letters in the string.
```

Thanks for adding an example too!